components/TableColumns/TableColumns.tsx

```diff
@@ -47,7 +62,7 @@ export const columns: ColumnDef<Header>[] = [
           variant="ghost"
           onClick={() => column.toggleSorting(column.getIsSorted() === 'asc')}
         >
-          HEADING
+          Entry #
```

Comment:
Why is this being hardcoded in? Would it make more sense to make it a prop so we can reuse this? Or maybe I'm misunderstanding the code.

Reply:
Answered in Discord by Alex.

> the reason the header titles are hardcoded is because when you render the table as <TableData data={whateverData} columns={differentColumns} /> <br /> <br /> the hard coded headers are the actual variants I can use. So there's columns for the league table, entry table, etc and then just columns={whateverColumns} for the type of table you want to do. <br /> <br /> I hardcode them because they get used and imported later on for each specific table variant <br /> <br /> that table columns file is where we will create a new table if we need and even define how many columns/rows it will get. <br /> <br /> TableData.tsx is where the layout and component of the table is built where you can import and use it wtih column variants. <br /> <br /> TableColumns.tsx is where you actually hardcode the specific Header titles you want for each specific table <br /> <br /> Because each column function code is the actual table functionality itself. So if Shashi wants to add something to the league table but not the entry table, then this would be easier to edit <br /> <br /> To help also, the part of the table with the three dots to add, edit, and delete is actually within the columns code. The columns code defines, renders, and creates the functionality for the table. The shadcn table itself is just how it gets laid out for the looks. <br /> <br /> So I created 3 seperate columns functions for the three separate tables we will need incase they need to be more unique.